### DISCUSSION TITLE
Adding LSP support for remapped Props (tailwindcss-intelisense)

### DISCUSSION THREAD

**Discussion by johnf** [nativewind/nativewind#706] [Show and tell]
I have a lot of custom components or remapped components where I create attributes like `containerClass`
If you are using tailwindcss-inteliphense that you can adjust classAttributes with a regex to get LSP completion on these other fields

I'm using nvim with lsp-config so I would o

```typescript
lspconfig.tailwindcss.setup({
  settings = {
    tailwindCSS = {
      classAttributes = { "class", "className", "[a-z]*ClassName" }
    }
  }
```

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
